capa/main.py

```diff
@@ -982,7 +982,11 @@ def main(argv=None):
                 return -1
 
     try:
-        sig_paths = get_signatures(args.signatures)
+        if args.format == "elf" or (args.format == "auto" and taste.startswith(b"\x7fELF")):
+            sig_paths = []
+            logger.debug("skipping library code matching: there are no ELF signatures yet")
+        else:
+            sig_paths = get_signatures(args.signatures)
```

Comment:
i think we should invert the condition: if its a PE file, then load signatures, otherwise don't. this allows us to add support for other file formats more easily, without adding a new condition here.

Reply:
yes, thanks